pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -581,9 +583,13 @@ spec:
     runNamespace: {{ $jivapodsns }}
     apiVersion: apps/v1
     kind: Deployment
-    objectName: {{ .Volume.owner }}-rep
     disable: {{ ne $shouldPatch "true" }}
     action: patch
+    repeatWith:
```

Comment:
this runTask seems to be adding all nodes details in all the deployments.. but, we shouldnot be doing it right?

Reply:
same comment applies for create-patchreplicadeployment as well

Reply:
fixed